aiomisc/compat.py

```diff
@@ -1,30 +1,12 @@
 import asyncio
 import logging
 import socket
-from functools import partial
-from typing import Any, Callable, Optional, TypeVar
+from contextvars import ContextVar
+from typing import Optional
 
 
 log = logging.getLogger(__name__)
 
-
-try:
-    # noinspection PyUnresolvedReferences
-    import contextvars
-
-    T = TypeVar("T")
-    F = TypeVar("F", bound=Callable[..., Any])
-
-    def context_partial(
-        func: F, *args: Any,
-        **kwargs: Any
-    ) -> Any:
-        context = contextvars.copy_context()
-        return partial(context.run, func, *args, **kwargs)
-except ImportError:
-    context_partial = partial   # type: ignore
-
-
 try:
     from time import time_ns
```

Comment:
time_ns was also added in python3.7, is this code necessary? Or some operating systems do not have such method (e.g. windows)?

Reply:
yes it's for windows